parser/test_nested/sub/deep.proto

```
@@ -0,0 +1,2 @@
+syntax = "proto3";
+package deep;
```

Buf (1.66.0)

[error] Files with package "deep" must be within a directory "deep" relative to root but were in directory "parser/test_nested/sub".

(PACKAGE_DIRECTORY_MATCH)